File: Implementation/Algorithms/python/Greedy/Ex01.py
# Problem: https://codeforces.com/problemset/problem/34/B

m, n = input().split()
arr = [int(i) for i in input().split()]

d = 0
c = 0
for i in arr:
    if c >= int(n):
        break

    if i < 0:
        d += i * -1
        c += 1

print(d)

File: Implementation/Algorithms/java/Greedy/Ex01.java
/*
 *
 * Problem: https://codeforces.com/problemset/problem/34/B
 *
 * */
import java.util.Scanner;

public class Ex01 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int m, n;
        int[] a = new int[10000];
        m = sc.nextInt();
        n = sc.nextInt();

        int d = 0;
        for (int i = 0; i < m; ++i) {
            a[i] = sc.nextInt();

            if (a[i] < 0)
                d += (a[i]) * -1;
        }

        System.out.println(d);
    }
}
